package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^7.18.0",
-    "ember-cli-htmlbars": "^4.2.3"
+    "ember-cli-htmlbars": "^5.3.1"
```

Comment:
Can you move this from deps to devDeps? This addon does not have any templates in `addon/` or `addon-test-support/`.

Reply:
Done!